float8_experimental/float8_dynamic_linear.py

```diff
@@ -73,47 +73,71 @@ def forward(self, x):
         return y
 
     @classmethod
-    def from_float(cls, mod, emulate: bool = False) -> "Float8DynamicLinear":
+    def create_meta_class(
+        cls, in_features: int, out_features: int
+    ) -> "Float8DynamicLinear":
```

Comment:
maybe a comment on why we're adding this?

Reply:
since we are removing float8_dynamic_linear I am going to remove the changes here